safeeyes/__main__.py

```diff
@@ -52,7 +51,9 @@ def __running():
             else:
                 # In older versions cmdline was a list object
                 cmd_line = proc.cmdline
-            if ('python3' in cmd_line[0] or 'python' in cmd_line[0]) and ('safeeyes' in cmd_line[1] or 'safeeyes' in cmd_line):
+            if ("python3" in cmd_line[0] or "python" in cmd_line[0]) and (
```

Comment:
What do you think about this format? It seems more readable (but require extra line of code)

```
if (
    (a or b)
    and (c or d)
):
```

Reply:
I agree that your formatting would be nicer. However, this is default behaviour of ruff (and black).
I don't want to differ from the standard formatting of black unless that is necessary.